spec/features/visitors/password_recovery_spec.rb

```diff
@@ -6,10 +6,11 @@
 #   So I can regain access to protected areas of the site
 feature 'Password Recovery' do
   def reset_password_and_sign_back_in(user)
-    fill_in 'New password', with: 'NewVal!dPassw0rd'
+    password = 'a really long password'
```

Comment:
shouldn't 'password' be on our not allowed list? either way, let's remember to follow up with NIST about their dictionary check